### PR TITLE
fetchArray connection parameter passed down through the Pool

### PR DESCRIPTION
**Pull request by ulmont**
See issue 199 https://github.com/markdirish/node-odbc/issues/199

The connection Pool doesn't persist a fetchArray parameter if one was passed in; this change adds fetchArray as one of the parameters the Pool will pass down to its new connections.

### CODE REVIEW

File: lib/Pool.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ const MAX_SIZE_DEFAULT = null;
 const SHRINK_DEFAULT = true;
 const CONNECTION_TIMEOUT_DEFAULT = 0;
 const LOGIN_TIMEOUT_DEFAULT = 0;
+const FETCH_ARRAY_DEFAULT = false; // RUC
```
Comment (kadler) [inline lib/Pool.js:17]: RUC?

Comment (ulmont) [reply lib/Pool.js:17]: Initialism in my working copy. I thought I removed all of those but seem to have missed one.

Comment (ulmont) [reply lib/Pool.js]: And corrected now.

Comment (kadler) [reply lib/Pool.js]: No problem. Figured it was something like that.

**Comment by kadler** [comment]
Can you squash the commits and add a signoff? DCO bot has more details: https://github.com/markdirish/node-odbc/pull/200/checks?check_run_id=2877356746

**Comment by ulmont** [comment]
> Can you squash the commits and add a signoff? DCO bot has more details: https://github.com/markdirish/node-odbc/pull/200/checks?check_run_id=2877356746

I think so.  Hopefully that worked as I'm definitely not a git expert.  I see it's listed as a single commit now though and the DCO error has gone away, so maybe.

**Review by kadler — APPROVED** [review]
LGTM

**Comment by markdirish** [comment]
The bad news: Now that the 2.4.0 branch has been merged into `master`, there are conflicts that must be resolved.
The good news: That now means that these changes will be included in 2.4.0

**Comment by markdirish** [comment]
LGTM!

**Pull request merged**